### ISSUE TITLE
Compact the emoji keyboard 

### ISSUE THREAD

**Issue by madushan1000** [AnySoftKeyboard/AnySoftKeyboard#1584]
The usual emoji keyboard has only 5 keys per row. In my opinion, this wastes a lot of screen realestate. 
![image](https://user-images.githubusercontent.com/4597144/50879946-bfb04080-1402-11e9-96ff-27a45fdd9642.png)

I prefer the 9 keys per row swiftkey style emoji keyboard.
![image](https://user-images.githubusercontent.com/4597144/50880005-ef5f4880-1402-11e9-832d-06416381fb4d.png)

But since that's my preferance, I would like to have an option to change the number of emoji keys per row from the preferances so users can set their prefered values. 

I would like to work on adding this feature, I forked the repo and curruntly going trough the code. Can somebody give me some pointers on what's the optimal way of achiving this?

**Comment by VitaliPom** [comment]
[CODE Dexamol]

I will try to take care of this soon, indeed did not understand what's so annoying in those emojies. Thanks for root causing this.

**Comment by jwsp1** [comment]
+1 on this issue :)

**Comment by jwsp1** [comment]
Removing the descriptions (Emoticons, people, accessories,...) would be a simple first step. Re-ordering seems to be more time-consuming

**Comment by AnXh3L0** [comment]
Yes, a very good point, we will keep this in mind for a major redesign in the near future.

**Comment by VitaliPom** [comment]
> Removing the descriptions (Emoticons, people, accessories,...) would be a simple first step. Re-ordering seems to be more time-consuming

Hi Julian, check out my version of Anysoftkeyboard, Effectedkeyboard2. 

https://play.google.com/store/apps/details?id=com.vitali.pom.effectedkeyboard2

**Comment by papb** [comment]
@VitaliPom Hi! Where can I find the repository for your version?

**Comment by VitaliPom** [comment]
> @VitaliPom Hi! Where can I find the repository for your version?

Hi, it's not Open Source yet. But you can see source of original parts in Menny's keyboard.

**Comment by jwsp1** [comment]
> Yes, a very good point, we will keep this in mind for a major redesign in the near future.

I'm very excited! Thanks :)

**Comment by jwsp1** [comment]
> > Removing the descriptions (Emoticons, people, accessories,...) would be a simple first step. Re-ordering seems to be more time-consuming
> 
> Hi Julian, check out my version of Anysoftkeyboard, Effectedkeyboard2.
> 
> https://play.google.com/store/apps/details?id=com.vitali.pom.effectedkeyboard2

I want to stick with ASK but thank you for sharing this app. Especially when @AnXh3L0 says that a redesign of some things is coming soon.

**Comment by moshpirit** [comment]
Do you know of any fork of ASK that implemented this already? It's a great expected feature :)
